.readthedocs.yaml

```diff
@@ -8,8 +8,13 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.11"
+    python: "3.12"
 
 # Build documentation in the "docs/" directory with Sphinx
 sphinx:
   configuration: docs/conf.py
+
+# Explicitly set the version of Python and its requirements
```

Comment:
The comment is not correct. Remove or replace by "Install required packages" or similar

Reply:
The comment comes directly from readthedoc doc: https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html#id1